searx/static/themes/simple/src/less/preferences.less

```diff
@@ -33,6 +33,21 @@ table {
     width: 300px;
   }
 
+  input[type="text"] {
+    width: 13.25rem;
+    color: var(--color-toolkit-input-text-font);
+    border: none;
+    background: none repeat scroll 0 0 var(--color-toolkit-select-background);
+    padding: 0.2rem 0.4rem;
+    height: 2rem;
+    .rounded-corners-tiny;
+
+    &:hover,
+    &:focus {
+      background-color: var(--color-toolkit-select-background-hover);
+    }
+  }
```

Comment:
> that's now solved by adding a new `.button` class 

then I assume, this can be (also) be moved back to line 55 / back into class `.value` .. right?